sdks/python/apache_beam/ml/inference/huggingface_inference.py

```diff
@@ -606,6 +603,11 @@ def __init__(
             task="text-generation", model="meta-llama/Llama-2-7b-hf",
             load_pipeline_args={'model_kwargs':{'quantization_map':config}})
 
+      device (str): the device on which you wish to run the pipeline. Defaults
```

Comment:
```suggestion
      device (str): the device (CPU or GPU) on which you wish to run the pipeline. Defaults
```

Note: applying this will break the format checker, so you should still probably do it locally